module/platform/Webots/src/Webots.cpp

```diff
@@ -309,4 +329,32 @@ void Webots::translate_and_emit_sensor(const SensorMeasurements& sensor_measurem
     }
 }
 
+// Converts the NUgus.proto servo name to the equivalent DarwinSensor.proto name
+DarwinSensors::Servo& translate_servo_id(const std::string& name, const DarwinSensors::Servos& servos) {
+    switch(name){
+        case "right_shoulder_pitch_sensor": return servos.rShoulderPitch;
+        case "left_shoulder_pitch": return servos.lShoulderPitch;
+        case "right_shoulder_roll": return servos.rShoulderRoll;
+        case "left_shoulder_roll": return servos.lShoulderRoll;
+        case "right_elbow_pitch": return servos.rElbow;
+        case "left_elbow_pitch": return servos.lElbow;
+        case "right_hip_yaw": return servos.rHipYaw;
+        case "left_hip_yaw": return servos.lHipYaw;
+        case "right_hip_roll": return servos.rHipRoll;
+        case "left_hip_roll": return servos.lHipRoll;
+        case "right_hip_pitch": return servos.rHipPitch;
+        case "left_hip_pitch": return servos.lHipPitch;
+        case "right_knee_pitch": return servos.rKnee;
+        case "left_knee_pitch": return servos.lKnee;
+        case "right_ankle_pitch": return servos.rAnklePitch;
+        case "left_ankle_pitch": return servos.lAnklePitch;
+        case "right_ankle_roll": return servos.rAnkleRoll;
+        case "left_ankle_roll": return servos.lAnkleRoll;
+        case "neck_yaw": return servos.headPan;
+        case "head_pitch": return servos.headTilt;
+
+        default: throw std::runtime_error("Out of bounds on servo name");
```

Comment:
Since we are switching on a string, is it correct to say "out of bounds"? 
```suggestion
        default: throw std::runtime_error("Invalid servo name");
```
Perhaps?

Reply:
Updated the error message. Also, we can't actually switch on strings.